dashboards/MongoDB/MongoDB_Oplog_Details.json

```diff
@@ -388,10 +388,12 @@
       "targets": [
         {
           "datasource": "Metrics",
-          "expr": "avg by (service_name) (mongodb_mongod_metrics_repl_buffer_count{service_name=~\"$service_name\"})",
+          "editorMode": "code",
```

Comment:
Is editorMode right here? Since it wasn't there before.

Reply:
Will this work for v2 ? or better   keep the only formula change?

Reply:
The file was exported from v2 only, and tested with v3 as well.
let me double check

Reply:
@matejkubinec Yes, its fine as it was included while exporting the dashboard from v2.